daml/Marketplace/Rule/AllocationAccount.daml

```diff
@@ -40,14 +40,15 @@ template AllocationAccountRule
       creditAccount : Account -> Asset -> Update (ContractId AssetDeposit)
       creditAccount = \account asset -> exerciseByKey @AssetSettlementRule account.id AssetSettlement_Credit with ctrl = nominee, ..
```

Comment:
Did you mean to take out the `AssetDeposit_SetObservers` code here?

Reply:
Nope. Added back